custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -159,7 +159,7 @@ def post_run(self, data_files):
         }
 
     def _consolidate_data(self, data_files):
-        result = defaultdict(dict)
+        result = defaultdict(OrderedDict)
```

Comment:
`dict` is insertion-ordered (like `OrderedDict`) [since Python 3.6 (officially 3.7)](https://stackoverflow.com/a/39980744)

Reply:
Ah yes, updating.